yt_dlp/extractor/weibo.py

```diff
@@ -187,13 +187,46 @@ class WeiboIE(WeiboBaseIE):
         'url': 'https://weibo.com/0/4224132150961381',
         'note': 'no playback_list example',
         'only_matching': True,
+    }, {
+        'url': 'https://m.weibo.cn/detail/5122647953771225',
+        'info_dict': {
+            'id': '5122647953771225',
+            'ext': 'mp4',
+            'display_id': '1034:5122647705714717',
+            'title': '阿呆与瓜的微博视频',
+            'duration': 164,
+            'timestamp': 1736817857,
+            'upload_date': '20250114',
+            'thumbnail': r're:https://.*\.jpg',
+            'view_count': int,
+            'like_count': int,
+            'repost_count': int,
```

Comment:
is this a test for a `mix_media_info` playlist? if so, please add as a playlist test (`'playlist_count': 1` etc)

Reply:
does this test actually pass? the playlist metadata is not  the same as the video metadata

Reply:
Well, actually I have not find how to call `_TESTS`.
`hatch test` does not test extractors.

from https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-support-for-a-new-site
> 6. Run `hatch test YourExtractor`

when I run `hatch test weibo`, got `ERROR: not found: /opt/github/yt-dlp/test/test_download.py::TestDownload::test_weibo`

Reply:
case matters, e.g. `hatch test Weibo`
but that would only run the first test case.
you would want to run `hatch test Weibo_all` or `hatch test Weibo_3`